src/pages/auth/login.jsx

```diff
@@ -0,0 +1,113 @@
+import "./auth.css";
+import { Link, useNavigate } from "react-router-dom";
+import { useState } from "react";
+import axios from "axios";
+import { useAuth } from "../../contexts/index";
+import { AiOutlineEyeInvisible, AiFillEye } from "react-icons/ai";
+export const Login = () => {
+  const [showPassword, setShowPassword] = useState(false);
+  const { setIsLoggedIn, setUserDetails } = useAuth();
+  const navigate = useNavigate();
+  const [loginCredentials, setLoginCredentials] = useState({
+    email: "",
+    password: "",
+  });
+  const [errorMsg, setErrorMsg] = useState("");
+  const guestCredentials = {
+    email: "adarshbalika@gmail.com",
+    password: "adarshbalika",
+  };
+  const setDetails = (event) => {
+    const name = event.target.name;
+    const value = event.target.value;
+    setLoginCredentials({ ...loginCredentials, [name]: value });
+  };
+  const loginHandler = async (data) => {
+    try {
+      const response = await axios.post("/api/auth/login", data);
+      if (response.status === 200) {
+        setIsLoggedIn(true);
+        setUserDetails(response.data.foundUser);
+        localStorage.setItem("token", response.data.encodedToken);
+        navigate("/");
+      }
+    } catch (error) {
+      setErrorMsg("YOUR EMAIL IS NOT REGISTERED WITH US😔");
+    }
+  };
+  const submitLoginForm = (event) => {
+    event.preventDefault();
+    loginHandler(loginCredentials);
+  };
+  return (
+    <section className="login flex-col-center">
+      <h2>Sign in</h2>
+      <form onSubmit={submitLoginForm} className="login-form flex-column">
+        <div className="input-field-wrapper flex-column">
+          <label for="email">Email</label>
```

Comment:
React Ignores For Attribute instead use htmlFor

Reply:
yup changes done